tests/integration/config/config.go

```diff
@@ -151,7 +161,21 @@ func GenerateTestCases() (testCases []*db.Case, testCasesMap *TestCasesMap) {
 				})
 
 				for _, p := range payloads {
-					testCasesMap.m[fmt.Sprintf("%s-%s-%s-%s-%s", ts, name, p, ph, enc)] = struct{}{}
```

Comment:
Maybe it is just me, but I don't understand such symbols economy: `p`, `ts`, `ph`, what is that? Without looking for cycles above it is not clear at all (btw there is not only one cycle - which can be ok maybe to name things such short for a one cycle, but 4 - it is a different story), `ts` - first thought of `timestamp`, but totally not `testset`, `p` - can be anything (never thought of `payload`). Just a thought. I think it is not a bad thing to use `set` or `payload` names.

Reply:
Fixed - https://github.com/wallarm/gotestwaf/pull/160/commits/b55e5367584402307d7fea6fbcac652d0e04b0b7